// it("should get an error using the handleErr function", async function () {});
